frontend/src/components/create-contract/PaymentSummary.tsx

```diff
@@ -1,3 +1,4 @@
+import { DAO_SERVICE_FEE } from '../../constants';
```

Comment:
The Dao Fee can change over time.  We can pull this from the `DAO_FEE` method on the smart contract.

```typescript
  const [daoFee, daeFeeLoading] = useSmartContractCall(
    contracts.Job.DAO_FEE
  );
  const daoFeeAsNumber = daoFee?.toNumber() || 0;
```

Reply:
It took me forever to add this gif, and you're right, done